NextcloudTalk/NCChatViewController.m

```diff
@@ -221,12 +234,31 @@ - (void)viewDidLoad
         self.navigationItem.compactAppearance = appearance;
         self.navigationItem.scrollEdgeAppearance = appearance;
         
-        [self.textInputbar setBackgroundColor:[UIColor secondarySystemBackgroundColor]];
-        [self.view setBackgroundColor:[UIColor secondarySystemBackgroundColor]];
+        [self.view setBackgroundColor:[UIColor systemBackgroundColor]];
+        [self.textInputbar setBackgroundColor:[UIColor systemBackgroundColor]];
+        
         [self.textInputbar.editorTitle setTextColor:[UIColor labelColor]];
+        [self.textView.layer setBorderWidth:1.0];
         [self.textView.layer setBorderColor:[UIColor systemGray4Color].CGColor];
     }
     
+    // Hide default top border of UIToolbar
+    [self.textInputbar setShadowImage:[UIImage new] forToolbarPosition:UIBarPositionAny];
+    
+    // Add new border subView to inputbar
+    self.inputbarBorderView = [UIView new];
+    [self.inputbarBorderView setAutoresizingMask:UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleBottomMargin];
+    self.inputbarBorderView.frame = CGRectMake(0, 0, self.textInputbar.frame.size.width, 1);
+    self.inputbarBorderView.hidden = YES;
+    
+    if (@available(iOS 13.0, *)) {
+        self.inputbarBorderView.backgroundColor = [UIColor systemGray6Color];
```

Comment:
We could also go with `systemGray5Color` here, as the border of the input is currently `systemGray4Color`

Reply:
I think it looks fine with `systemGray6Color` :)